src/property-filter/interfaces.ts

```diff
@@ -140,6 +140,15 @@ export interface PropertyFilterProps extends BaseComponentProps, ExpandToViewpor
    * Specifies the maximum number of displayed tokens. If the property isn't set, all of the tokens are displayed.
    */
   tokenLimit?: number;
+  /**
+   * Label that will be passed down to the Autosuggest `ariaLabel` property.
+   * See the [Autosuggest API](/system/components/autosuggest/?tabId=api) page for more details.
```

Comment:
is the `/system/` part in the URL intentional? I did not notice that before in our code, and would have expected the URL to just be `/components/autosuggest/?tabId=api`